tests/spread/tutorials/basic/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: smoke test for Chisel
+
+execute: |
+  set -eux
```

Comment:
That shouldn't be necessary.  Spread will do the right things already IIRC.

Reply:
Ack. Removed